### PR TITLE
Create logging namespace and operatorgroup on stone-stage-p01

### PR DESCRIPTION
**Pull request by hugares**
All the clusters we had so far were ROSA classic, i.e. control plane nodes part of the cluster nodes. On those clusters, openshift-logging namespace and operatorgroup are already created and managed by hive.

The new stone-stage-p01 is a ROSA HCP(Hosted Control Plane) cluster and on those, the openshift-logging namespace and operatorgroup are not created/managed.

Move staging and production kustomize files into nested base folders and change logging argocd application to use base as default clusterDir. On stone-stage-p01, override the cluster dir to point to staging/stone-stage-p01 which include the base plus namespace and operatorgroup.

Also set argocd syn waves of a few resources to make sure the are created in the right order.

[RHTAPWATCH-695](https://issues.redhat.com//browse/RHTAPWATCH-695)

### CODE REVIEW

**Review by amisstea — APPROVED** [review]
LGTM

**Review by eisraeli — APPROVED** [review]
LGTM. IIRC, we'll also need to apply the same configuration for stone-prod-p01, right ?

**Review by eisraeli — APPROVED** [review]
Great ! LGTM.
We'll also need to apply the same configuration for stone-prod-p01, right ?

File: components/monitoring/logging/base/configure-logging/configure-log-collectors.yaml
Comment (hugares) [reply components/monitoring/logging/base/configure-logging/configure-log-collectors.yaml:9]: I was not able to test them, I am relying on dev environment and knowledge of what needs to be created before what.

**Comment by hugares** [comment]
> LGTM. IIRC, we'll also need to apply the same configuration for stone-prod-p01, right ?

yes, but stone-prod-p01 does not exist yet. That said, we can still create the configuration for it right away. Do you want me to add it to this PR or I do another one?

**Comment by eisraeli** [comment]
> > LGTM. IIRC, we'll also need to apply the same configuration for stone-prod-p01, right ?
> 
> yes, but stone-prod-p01 does not exist yet. That said, we can still create the configuration for it right away. Do you want me to add it to this PR or I do another one?

We're good creating it on a different PR once the cluster exists.

File: components/monitoring/logging/base/configure-logging/configure-log-collectors.yaml
Comment (eisraeli) [reply components/monitoring/logging/base/configure-logging/configure-log-collectors.yaml:9]: Alright.

**Comment by hugares** [comment]
@eisraeli Can you /lgtm the PR if we are ready to get it in?

**Comment by dheerajodha** [comment]
The [latest](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-appstudio_infra-deployments/3012/pull-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-tests/1744843188067110912) `ci/prow/appstudio-e2e-tests` job failed on [STONEINTG-742](https://issues.redhat.com/browse/STONEINTG-742) whose fix was already merged to prod via [this PR](https://github.com/redhat-appstudio/infra-deployments/pull/3019), please rebase [your branch](https://github.com/hugares/infra-deployments/tree/logging-private)

**Comment by hugares** [comment]
@eisraeli I had to rebase, I will need /lgtm again

**Pull request merged**
